src/pudl/metadata/resources/eia860.py

```diff
@@ -846,12 +878,15 @@
         "etl_group": "eia860",
     },
     "core_eia860__scd_generators_energy_storage": {
-        "description": (
-            "Annually reported information about energy storage from EIA-860 Schedule 3."
-            " This table includes only those values that are unique to energy storage. "
-            "The rest of the columns that are reported in the EIA-860 Energy Storage tabs are "
-            "included in core_eia860__scd_generators and core_eia__entity_generators."
-        ),
+        "description": {
+            "additional_summary_text": "energy storage which may change from year to year.",
```

Comment:
Very little nit, but you say "vary from year to year" in all other cases (vs. change).